content_zh/docs/concepts/traffic-management/load-balancing.md

```diff
@@ -16,7 +16,7 @@ keywords: [traffic-management,load-balancing]
     caption="发现与负载均衡"
     >}}
 
-如上图所示，网格中的服务使用其 DNS 名称访问彼此。服务的所有 HTTP 流量都会通过 Envoy 自动重新路由。Envoy 在负载均衡池中的实例之间分发流量。虽然 Envoy 支持多种[复杂的负载均衡算法][https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/load_balancing]，但 Istio 目前仅允许三种负载平衡模式：轮循、随机和带权重的最少请求。
+如上图所示，网格中的服务使用其 DNS 名称访问彼此。服务的所有 HTTP 流量都会通过 Envoy 自动重新路由。Envoy 在负载均衡池中的实例之间分发流量。虽然 Envoy 支持多种[复杂的负载均衡算法](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/load_balancing)，但 Istio 目前仅允许三种负载平衡模式：轮循、随机和带权重的最少请求。
```

Comment:
For future PR, it is better to create such PR separately for better tracing.

Reply:
OK, I will send PR for files one by one.